.circleci/config.yml

```diff
@@ -35,6 +35,22 @@ jobs:
       - run: npm run lint
       - run: npx commitlint --from origin/master
 
+  docs:
+    <<: *defaults
+    docker:
+      - image: circleci/python:3.6-node-browsers
+    steps:
+      - checkout
+      - <<: *install
+      - run:
+          working_directory: docs
+          command: pipenv install --dev
```

Comment:
currently the only dev dependency is autobuild, which isn't needed on CI, but depends whether you plan to add more dev deps

Reply:
I wanted to look into linting and such, spell checking etc so plan was to iterate on that afterwards.